app/views/group_offers/index.html.slim

```diff
@@ -45,7 +52,7 @@ table.table.table-striped
                 br
         td= t("offer_state.#{offer.offer_state}") if offer.offer_state?
         td
-          - if offer.responsible
+          - if offer.responsible && !offer.external?
```

Comment:
the other way arround because its cheaper: `!offer.external? && offer.responsible`